packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -0,0 +1,343 @@
+            // Arrange
+            const date = moment(today);
+            const defaultValue = date.format("YYYY-MM-DD");
+
+            // Act
+            render(
+                <BirthdayPicker
+                    defaultValue={defaultValue}
+                    onChange={() => {}}
+                />,
+            );
+
+            // Assert
+            expect(
+                screen.getByTestId("birthday-picker-month"),
+            ).toHaveTextContent("Jul");
+            expect(screen.getByTestId("birthday-picker-day")).toHaveTextContent(
+                "19",
+            );
+            expect(
+                screen.getByTestId("birthday-picker-year"),
+            ).toHaveTextContent("2021");
+            expect(screen.queryByRole("alert")).not.toBeInTheDocument();
+        });
+
+        it("renders with a invalid default future value", () => {
+            // Arrange
+            jest.useRealTimers();
+            DateMock.advanceTo(today);
+            const date = moment(today).add(1, "day");
+            const defaultValue = date.format("YYYY-MM-DD");
+
+            // Act
+            render(
+                <BirthdayPicker
+                    defaultValue={defaultValue}
+                    onChange={() => {}}
+                />,
+            );
+
+            // Assert
+            expect(
+                screen.getByTestId("birthday-picker-month"),
+            ).toHaveTextContent("Jul");
+            expect(screen.getByTestId("birthday-picker-day")).toHaveTextContent(
+                "20",
+            );
+            expect(
+                screen.getByTestId("birthday-picker-year"),
+            ).toHaveTextContent("2021");
+        });
+
+        it("renders an error with a invalid default future value", async () => {
+            // Arrange
+            const date = moment(today).add(1, "day");
+            const defaultValue = date.format("YYYY-MM-DD");
+
+            // Act
+            render(
+                <BirthdayPicker
+                    defaultValue={defaultValue}
+                    onChange={() => {}}
+                />,
+            );
+
+            // Assert
+            expect(screen.getByRole("alert")).toBeInTheDocument();
+        });
+
+        it("renders with an invalid default value", () => {
+            // Arrange
+            const defaultValue = "2021-02-31"; // There is no Feb 31st
+
+            // Act
+            render(
+                <BirthdayPicker
+                    defaultValue={defaultValue}
+                    onChange={() => {}}
+                />,
+            );
+
+            // Assert
+            expect(
+                screen.getByTestId("birthday-picker-month"),
+            ).toHaveTextContent("Month");
+            expect(screen.getByTestId("birthday-picker-day")).toHaveTextContent(
+                "Day",
+            );
+            expect(
+                screen.getByTestId("birthday-picker-year"),
+            ).toHaveTextContent("Year");
```

Comment:
Can you break the `screen.getByTestId` calls out into their own thing, possibly in the Act section so that the expectations are easier to parse?

Aside: This seems like a good example of where multiple expectations kinda make sense. However, I wonder if instead of getting each part of the picker, we took a holistic view and did `toHaveTextContent("MonthDayYear")` on the picker itself - would that work even?

Hmm, I dunno. I'm just riffing. No change requested.